CHANGELOG.md

```diff
@@ -10,19 +16,6 @@
 ### Fixes
 
 - Remove empty session replay tags (#4667)
-- - `SentrySdkInfo.packages` should be an array (#4626)
-- Use the same SdkInfo for envelope header and event (#4629)
-
-### Improvements
-
-- Improve compiler error message for missing Swift declarations due to APPLICATION_EXTENSION_API_ONLY (#4603)
-- Mask screenshots for errors (#4623)
-- Slightly speed up serializing scope (#4661)
-
-### Internal
-
-- Remove loading `integrations` names from `event.extra` (#4627)
-- Add Hybrid SDKs API to add extra SDK packages (#4637)
```

Comment:
I think I resolved this conflict incorrectly. Did we want these items from the beta release's changes repeated in the 8.43.0 RC?

Reply:
You are correct, there have been a couple of releases since I opened up this PR. I'll fix this when getting ready to merge.